kursinis.tex

```diff
@@ -27,7 +27,7 @@
 \author{Vardenis Pavardenis}
 % \secondauthor{Vardonis Pavardonis}   % Pridėti antrą autorių
 \supervisor{prof. habil. dr. Vardaitis Pavardaitis}
-\addsignatureplaces{} % prideda parašų vietas tituliniame puslapyje
+% \addsignatureplaces{} % prideda parašų vietas tituliniame puslapyje

```

Comment:
Ar nebūtų geriau, jeigu parašų vietos būtų pridėtos pagal numatymą?

Reply:
Gal, bet dabar bent jau kursiniai darbai nebėra spausdinami, tai kaip ir beprasmiška šitą turėti.

Reply:
Nors nurodymuose pateiktame pavyzdyje parašų vietos vis dar yra.

Reply:
Tai gal kursiniam darbui galime nuimti, o bakalauriniam darbui kol kas palikti parašus įjungtus? Manau būtų geras kompromisas.